antismash/common/secmet/record.py

```diff
@@ -89,6 +89,7 @@ def __setattr__(self, attr: str, value: Any) -> None:
         except AttributeError:
             raise AttributeError("Record does not support dynamically adding attributes")
 
+    @property
```

Comment:
is_circular was a method. You were testing if the method is None, not the results of the method. To be consistent with other code in the codebase, the correct approach would be to call the method itself by adding the parentheses.

Reply:
Understood, thanks.. Reverting. Is this method being used anywhere else in the code so far? I could not find it but perhaps i missed it?

Reply:
This one in particular isn't, as you have changed the lines that used it, but the intended style is that anything with a verb in the name is a function (e.g. `has_complement(), is_contained_by(), overlaps_with()`) or a method, while simple nouns would be properties or plain members (e.g. `.cds_children, .location, .results`).